src/pages/groupComparison/GroupComparisonURLWrapper.ts

```diff
@@ -0,0 +1,33 @@
+import URLWrapper from "../../shared/lib/URLWrapper";
+import ExtendedRouterStore from "../../shared/lib/ExtendedRouterStore";
+import {computed} from "mobx";
+import {getTabId, GroupComparisonTab} from "./GroupComparisonUtils";
+import autobind from "autobind-decorator";
+import {OverlapStrategy} from "./GroupComparisonStore";
+
+export type GroupComparisonURLQuery = {
+    sessionId: string;
+    groupOrder?:string; // json stringified array of names
+    unselectedGroups?:string; // json stringified array of names
+    overlapStrategy?:OverlapStrategy;
+};
+
+export default class GroupComparisonURLWrapper extends URLWrapper<GroupComparisonURLQuery> {
+    constructor(routing:ExtendedRouterStore) {
+        super(routing, [
+            { name:"sessionId", persisted: false },
```

Comment:
I wonder if "sessionProp" would be more descriptive?   persisted is kind of misleading because we DO want to persist all params, just not in a remote session.

Reply:
`sessionProp` sounds pretty good to me